apps/src/lab2/projects/utils.ts

```diff
@@ -1,11 +1,12 @@
 import getScriptData from '@cdo/apps/util/getScriptData';
-import {ProjectFile, ProjectType} from '../types';
+import {ProjectFile} from '../types';
 
 // Partial definition of the App Options structure, only defining the
 // pieces we need in this component.
 interface PartialAppOptions {
```

Comment:
Should `AppOptions` also be updated in `lab2/types`?

Reply:
Hmm good question, we don't even use the `AppOptions` type in `lab2/types` as far as I'm aware, so maybe we just need to move this interface into `lab2/types`

Reply:
> we don't even use the AppOptions type in lab2/types as far as I'm aware

Interesting! So it's more for documentation? I was wondering if we could use `Omit` like we did [here](https://github.com/code-dot-org/code-dot-org/blob/731456039cdbb9b61e11b5c83d761f379d483aa1/apps/src/aichat/types.ts#L74), but I see that in `AppOptions` we use `level: Level` instead of `levelId: number`.

Reply:
Yeah, that `AppOptions` type was added in the early days of Lab2 - our philosophy since then has been to try to use AppOptions as little as possible, which is why we have this `PartialAppOptions` where we only define the properties we use.